cinema/models.py

```diff
@@ -35,12 +39,21 @@
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(instance: "Movie", filename: str) -> pathlib.Path:
+    filename = (
+        f"{slugify(instance.title)}-{uuid.uuid4()}"
+        + pathlib.Path(filename).suffix
+    )
+    return pathlib.Path("uploads/movies") / pathlib.Path(filename)
```

Comment:
The `movie_image_path` function is correctly defined to generate a unique file path for movie images using the movie title and a UUID.